cache/lru_cache.h

```diff
@@ -487,6 +487,9 @@ class LRUCache
   size_t TEST_GetLRUSize();
   //  Retrieves high pri pool ratio.
   double GetHighPriPoolRatio();
+  std::shared_ptr<SecondaryCache> GetSecondaryCache() {
```

Comment:
This should return a const reference and be marked const

Reply:
This line is removed in the new commit.